specs/latest/1.0/Makefile

```diff
@@ -7,4 +7,4 @@ webgl.idl: index.html extract-idl.py Makefile
 	echo "//" >> webgl.idl
 	echo "// WebGL IDL definitions scraped from the Khronos specification:" >> webgl.idl
 	echo "// https://www.khronos.org/registry/webgl/specs/latest/" >> webgl.idl
-	PYTHONPATH=../../../resources/html5lib/src python extract-idl.py index.html >> webgl.idl
```

Comment:
I'm guessing that when this script was updated to python3, the author ran it directly (with system-installed dependencies). It's not really obvious that it's supposed to be run through the Makefile instead, especially since it's executable and has a shebang. What do you think about either `chmod -x extract-idl.py` so that it's not executable directly on \*nix (and/or changing the shebang to `#!/usr/bin/false # Run this via Makefile instead`) or printing a warning from extract-idl.py if PYTHONPATH is not set?

Reply:
It's possible that someone rebuilt the IDL having had those dependencies installed locally. From talking with you and @greggman just now, it sounds like guaranteeing a hermetic environment is a little tricky. For now I'd like to merge this so that the IDL is updated, and will look into adding a check to the Github actions to ensure the IDL is up to date with every pull request. That should help.